docs/tactics.md

```diff
@@ -169,9 +169,21 @@ they are only meant to be used on small, straightforward problems.
 All accept an optional list of simplifier rules, typically definitions that should be expanded.
 (The equations and identities should not refer to the local context.) All also accept an optional list of `ematch` lemmas, which must be preceded by `using`.
 
+### abel
+
+Evaluate expressions in the language of *additive*, commutative monoids and groups.
```

Comment:
Could you add another sentence (and an example) about the `at` specifier?

Reply:
Added.